lib/internal/assert.js

```diff
@@ -10,13 +10,18 @@ let green = '';
 let red = '';
 let white = '';
 
-const READABLE_OPERATOR = {
-  deepStrictEqual: 'Input A expected to strictly deep-equal input B',
-  notDeepStrictEqual: 'Input A expected to strictly not deep-equal input B',
-  strictEqual: 'Input A expected to strictly equal input B',
-  notStrictEqual: 'Input A expected to strictly not equal input B'
+const kReadableOperator = {
+  deepStrictEqual: 'Expected input to be strictly deep-equal',
```

Comment:
suggestion: "inputs" instead of "input"

Reply:
I am good either way as it seems that both would work. @Trott @vsemozhetbyt do you have a preference?

Reply:
For me, "inputs" seems more clear (no confusing "input is deep-equal — to what?"), but I am not a native speaker)

Reply:
I agree with @targos and @vsemozhetbyt for the reason offered by @vsemozhetbyt (using "inputs" removes potential ambiguity).